src/services/event/event-router.ts

```diff
@@ -436,6 +436,58 @@ eventsRouter.post("/", strongJwtVerification, async (req: Request, res: Response
 	}
 });
 
+/**
+ * @api {get} /event/expiration/:EVENTID GET /event/expiration/:EVENTID
+ * @apiGroup Event
+ * @apiDescription Get the expiration time for requested event.
+ * 
+ * @apiParam {String} EVENTID The unique identifier of the event.
+ *
+ * @apiSuccess (200: Success) {Json} event The existing event and expiration data.
+ * @apiSuccessExample Example Success Response:
+ * HTTP/1.1 200 OK
+ * {
+ *    "id": "52fdfc072182654f163f5f0f9a621d72",
+ *    "exp": 1532202702,
+ * }
+ * @apiUse strongVerifyErrors
+ * @apiError (403: Forbidden) {String} InvalidPermission Access denied for invalid permission.
+ * @apiErrorExample Example Error Response:
+ *     HTTP/1.1 403 Forbidden
+ *     {"error": "InvalidPermission"}
+ * @apiError (400: Bad Request) {String} InvalidParams Invalid parameters for the event.
+ * @apiErrorExample Example Error Response:
+ *     HTTP/1.1 400 Bad Request
+ *     {"error": "InvalidParams"}
+ * @apiError (500: Internal Error) {String} InternalError Database operation failed.
+ * @apiErrorExample Example Error Response:
+ *     HTTP/1.1 500 Internal Server Error
+ *     {"error": "DatabaseError"}
```

Comment:
Keep consistent within apiErorr and apiErrorExample

DatabaseError -> InternalError